### PR TITLE
update articles

### PR DESCRIPTION
**Pull request by wmccone**
Added a exclaimation point!

### CODE REVIEW

**Comment by rreinhardt9** [comment]
@wmccone How's this going?

File: app/views/articles/new.html.erb [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<h1 class="text-center mt-4">Create new article </h1>
+<h1 class="text-center mt-4">Create new article! </h1>
```
Comment (rreinhardt9) [inline app/views/articles/new.html.erb:1]: This looks much more exciting!

File: app/views/articles/new.html.erb [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
```
Comment (wmccone) [inline app/views/articles/new.html.erb:2]: @rreinhardt9 this is a great feature!

Comment (rreinhardt9) [reply app/views/articles/new.html.erb:2]: It's so nice!!

**Pull request closed without merging**